mmv1/products/compute/VpnGateway.yaml

```diff
@@ -126,6 +132,17 @@ properties:
       The network this VPN gateway is accepting traffic for.
     required: true
     immutable: true
+  - !ruby/object:Api::Type::Enum
+    name: 'stack_type'
+    description: |
+      The stack type for this VPN gateway to identify the IP protocols that are enbaled.
+      If not specified, IPV4_ONLY will be used.
+    default_value: :IPV4_ONLY
+    values:
+      - :IPV4_ONLY
+      - :IPV4_IPV6
+    immutable: true
+    ignore_read: true    
```

Comment:
Why do we ignore read here? Is this value not returned by the api?

Reply:
You are right. The API does return the stackType. Removed ignore